projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.interface.ts

```diff
@@ -252,7 +258,7 @@ export interface IPivotGridRecord {
     /** List of dimensions associated with the record.**/
     dimensions: IPivotDimension[];
     /** Describes if this is a total record for a dimension */
-    totalRecord?: boolean;
+    totalRecordDimName?: string;
```

Comment:
The description was kinda okay for the bool prop, but certainly doesn't explain what this is now. Also `Dim` might be fine for internal API but none of the public ones abbreviate like that, so the comment and perhaps the name need an update.

Reply:
updated